packages/core/stories/input/input.doc.stories.mdx

```diff
@@ -48,6 +50,8 @@ Input has two variants: primary (default) and secondary.
   <Story id="core-input--variants" />
 </Canvas>
 
+- For recommendations on variant choice, see the Forms pattern.
```

Comment:
Could this link to the Forms pattern?

Reply:
done :) i avoided adding it to storybook bc of links potentially changing on the site but should be ok